pkg/objectupdate/rte/rte_test.go

```diff
@@ -74,7 +74,7 @@ func TestUpdateDaemonSetArgs(t *testing.T) {
 			name: "defaults",
 			conf: nropv1.DefaultNodeGroupConfig(),
 			expectedArgs: []string{
-				"--pods-fingerprint", "--refresh-node-resources", "--sleep-interval=10s", "--notify-file=/run/rte/notify",
+				"--pods-fingerprint", "--pods-fingerprint-unrestricted=false", "--refresh-node-resources", "--sleep-interval=10s", "--notify-file=/run/rte/notify",
```

Comment:
Another reason I am not a fan of this name is because it inherently has a negative in the name which makes this confusing. I would prefer it it was `--pods-fingerprint-exclusive-resource-only=true` making the code more readable and clear on what is expected here.

Reply:
point taken!